src/geo.c

```diff
@@ -552,17 +634,28 @@ void georadiusGeneric(client *c, int flags) {
         return;
     }
 
+    if ((flags & GEOSEARCH) && !(frommember || fromloc)) {
+        addReplyErrorFormat(c, "%s FROMMEMBER and FROMLOC "
+                         "cannot exist at the same time but one must exist", (char *)c->argv[0]->ptr);
```

Comment:
nitpick: "exactly one of FROMMEMBER or FROMLOC can be specified for %s" is slightly more concise.

It would also be nice to throw this exception when you provide both, as opposed to the syntax error you currently throw.

Reply:
I mean FROMMEMBER and FORMLOC, they cannot exist at the same time, but one of them must exist.

Thanks for your suggestions, I will update.